ibm/resource_ibm_kms_key_test.go

```diff
@@ -125,7 +137,7 @@ func testAccCheckIBMKmsResourceRootkeyWithCOSConfig(instanceName, KeyName, cosIn
 		key_protect          = ibm_kms_key.test.id
 	}
 	
-`, instanceName, KeyName, cosInstanceName, bucketName)
+`, instanceName, KeyName, expirationdate, cosInstanceName, bucketName)
 }
```

Comment:
Don't change all existing testcase because we are missing the flow of expiration  nil

Reply:
Which ones should I revert?

Reply:
Or would I need to write my own test cases?

Reply:
Let the old testcase remain same..write a new testcase

Reply:
Ok.